samples/GraphQL.DataLoader.Example/Program.cs

```diff
@@ -0,0 +1,43 @@
+using DataLoaderGql;
+using DataLoaderGql.GraphQl;
+using GraphQL;
+using Microsoft.EntityFrameworkCore;
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddScoped<SalespeopleByNameDataLoader>().AddScoped<CarsBySalespersonDataLoader>();
+// Add services to the container.
+// Learn more about configuring OpenAPI at https://aka.ms/aspnet/openapi
+builder.Services.AddDbContext<DealershipDbContext>();
+
+builder.Services.AddGraphQL(builder =>
+{
+    builder
+        .AddSystemTextJson()
+        .AddGraphTypes(typeof(CarsGraphType).Assembly)
+        .AddSchema<DealershipSchema>();
+});
+var app = builder.Build();
+
+app.UseGraphQL();
+app.UseGraphQLAltair();
+app.MapGraphQLAltair().AllowAnonymous();
+await app
+    .Services
+    .CreateAsyncScope()
+    .ServiceProvider
+    .GetRequiredService<DealershipDbContext>()
+    .Database
+    .MigrateAsync()
+    .ConfigureAwait(false);
+
+await app
+    .Services
+    .CreateAsyncScope()
+    .ServiceProvider
+    .GetRequiredService<DealershipDbContext>()
+    .Database
+    //For making sure seed Data gets applied
+    .EnsureCreatedAsync()
+    .ConfigureAwait(false);
```

Comment:
I'm not sure why both `Migrate` and `EnsureCreated` are in use, but chances are that either (a) one can be eliminated, or (b) we can just add this to the above:

```cs
await dbContext.Database.EnsureCreatedAsync().ConfigureAwait(false);
```

If both are required, it would seem that EnsureCreated would have to come first ????  But I could be wrong.

Reply:
i call ensureCreated only for the Seeding to run. This is something they seemingly changed in EFCore 9, where EnsureCreated actually didn't Create the DB Schema for me anymore and told me to migrate. I can try if Migrate also calls the Seeding Methods, but i think it does sadly not.

Reply:
i guess it was both other things i had messed up in the middle of building this. We have free choice of which one we want to use. So i guess we go with EnsureCreated, because it's the simpler version and we can leave out the Migrations.

Reply:
I removed the Migrations and left it with EnsureCreated with your suggested Style from above

Reply:
Sounds great!  Makes the sample simpler, and keeps it focused on demonstrating the data loader, not demonstrating EF.